user_docs/en/changes.t2t

```diff
@@ -31,9 +31,11 @@ Windows 8.1 is the minimum Windows version supported.
 - Added support for Bluetooth Low Energy HID Braille displays. (#15470)
 - A new "on-demand" speech mode has been added.
 When speech is on-demand, NVDA does not speak automatically (e.g. when moving the cursor) but still speaks when calling commands whose goal is explicitly to report something (e.g. report window title). (#481, @CyrilleB79)
+- it is now possible to exclude unwanted speech modes when cycling between them using ``NVDA+s`` from the Speech category in NVDA's settings. (#15806, @lukaszgo1)
+  - If you are currently using the NoBeepsSpeechMode add-on consider uninstalling it ,and disabling "beeps" and "on-demand" modes in the settings.
+  -

```

Comment:
I don't think mentioning the add-on is relevant in the core change log. We haven't done this historically when other add-ons have been integrated (e.g. add-on store, OCR).
I've also move the sentence around to be clearer.

```suggestion
- In the NVDA's settings Speech category, it is now possible to exclude unwanted speech modes when cycling between them using ``NVDA+s``. (#15806, @lukaszgo1)
```

Reply:
Well, I just tried, and it apparently gives no errors, but hides completely the new functionality and, moreover, the new on-demand speech mode.

Reply:
note that the add-on will be marked incompatible anyway, so I'm not sure whether it should be addressed. People should no longer update it.

Reply:
I mentioned add-on in the change log since:
- it is pretty popular
- it has even been mentioned in the NV Access training materials, and I'm not sure how easy is to update them (especially the audiobook and printed Braille copy), therefore mentioning that NoBeepsSpeechMode is unnecessary somewhere which is localized seemed a right thing to do
- With the newly added ability to override compatibility for add-ons the fact that add-on is no longer compatible is much less important. Most users would override compatibility if no updated version is available, so mentioning in the change log that there is a core function which should be used instead is the least we can do to discourage forcefully enabling the add-on


Personally I think we may consider going even a step further i.e. detect NoBeepsSpeechMode in the core, if its enabled let user know that it is no longer needed, offer to configure NVDA in the same way it behaved with the add-on and uninstall it. While that would be a new paradigm in NVDA I'd say it is worth considering.